BoxContentSDK/BoxContentSDK/BOXURLSessionManager.m

```diff
@@ -448,7 +463,7 @@ - (void)cancelAndCleanUpBackgroundSessionTasksForUserId:(NSString *)userId error
 {
     //FIXME: make sure set up is completed, and prevents any new tasks created for this userId while we clean it up
 
-    if (userId != nil) {
+    if (self.supportBackgroundSessionTasks == YES && userId != nil) {
```

Comment:
this only applies if we support background session tasks, if user never sets up to support background tasks, there will be nothing to clean.